wicket/src/runner.rs

```diff
@@ -345,3 +356,52 @@ async fn run_event_listener(
         }
     });
 }
+
+struct EventReportLogThrottler {
+    last_log: Option<Instant>,
+    min_time_between_logs: Duration,
+}
+
+impl Default for EventReportLogThrottler {
+    fn default() -> Self {
+        const DEFAULT_TIME_BETWEEN_LOGS: Duration = Duration::from_secs(60);
+        Self::new(DEFAULT_TIME_BETWEEN_LOGS)
```

Comment:
I suspect 60s is a long time here. Can this be brought down to eg 15s?

Reply:
Sure, I wasn't sure what was reasonable here. Even with the report size correct these still end up pretty large - a few hundred lines, multiplied by however many sleds have relevant reports (from running or completed updates). We can try 15s and see how it is though!